calibration/CTDMOG/CGINS-CTDMOG-13596__20170327.csv

```diff
@@ -10,11 +10,11 @@ serial,name,value,notes
 37-13596,CC_i,-1.580195e-004,
 37-13596,CC_j,3.119809e-005,
 37-13596,CC_p_range,1450,
-37-13596,CC_pa0,1.139298e+000,
-37-13596,CC_pa1,4.570067e-003,
-37-13596,CC_pa2,-2.952698e-011,
-37-13596,CC_ptca0,5.247850e+005,
-37-13596,CC_ptca1,6.367859e+000,
+37-13596,CC_pa0,1.189856e+00,
+37-13596,CC_pa1,4.571489e-03,
+37-13596,CC_pa2,-3.384865e-11,
+37-13596,CC_ptca0,5.246455e+05,
+37-13596,CC_ptca1,6.367858e+00,
 37-13596,CC_ptca2,2.146824e-001,
 37-13596,CC_ptcb0,2.523850e+001,
 37-13596,CC_ptcb1,-3.000000e-004,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT